notebooks/time-to-merge-prediction/seldon-deployment-config.yaml

```diff
@@ -1,15 +1,51 @@
 apiVersion: machinelearning.seldon.io/v1alpha2
 kind: SeldonDeployment
 metadata:
-  name: ttm-sklearn-pipeline
+  name: github-pr-ttm-seldon-deployment
 spec:
-  name: ttm-model-pipeline
+  name: github-pr-ttm-deployment
   predictors:
-  - graph:
+  - componentSpecs:
+    - spec:
+        containers:
+        - image: quay.io/kachau/github-pr-ttm-seldon:latest
```

Comment:
so this needs to be replaced by upstream once this PR is merged and a release is tagged?